### PR TITLE
Change pip version in setup.py

### PR DESCRIPTION
**Pull request by zurk**
it works well and all Travis tests are passed.

It is update of https://github.com/src-d/ml/pull/199

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
                       "humanize>=0.5.0",
                       "parquet>=1.2,<2.0",
                       "pygments>=2.2.0,<3.0",
-                      "pip==9.0.1"] + typing,
+                      "pip==9.0.3"] + typing,
```
Comment (vmarkovtsev) [inline setup.py]: Please change it to `pip!=9.0.2`

Comment (zurk) [reply setup.py]: also it reqierements.txt?

**Comment by vmarkovtsev** [comment]
@zurk This is weird, there is no Travis build

**Comment by zurk** [comment]
Link to travis build. It is failed to pin automatically https://travis-ci.org/src-d/ml/builds/356816856

**Pull request merged**